docs/howtoguides/enable_realtime_kernel.rst

```diff
@@ -1,92 +1,147 @@
 .. _manage-realtime:
 
-How to manage real-time kernel
+How to manage Real-time Ubuntu
```

Comment:
Hi @s-makin !

Is there a reason for using "How to _manage_ Real-time Ubuntu" instead of "How to _install_ Real-time Ubuntu"?
The rest of the set up is really related to Pro but not really "managing" RTU

Also I think you may need to rename the title in the main navigation menu. it currently says "real-time kernel" but you've updated the page title to "Real-time Ubuntu".

Thanks!